Makefile

```diff
@@ -307,6 +329,30 @@ karma: build
 .PHONY: build
 build: add-on standalone ui vendor_libs
 
+GIT_EXPORT_LOCATION := ../gecko-dev
+GIT_EXPORT_DIR := $(GIT_EXPORT_LOCATION)/browser/extensions/loop
+
+.PHONY: git-export
+git-export: build dist_export
+	find -E $(GIT_EXPORT_DIR) -type f ! -regex \
+	  '.*/(moz.build|README.txt|.gitignore|run-all-loop-tests.sh|manifest.ini)' -delete
+	$(RSYNC) $(DIST_EXPORT_DIR)/* $(GIT_EXPORT_DIR)
+	# TEMPORARY ITEMS to get l10n to work the "traditional" way in m-c whilst we
+	# sort out a proper fix.
+	# Drop locales from the jar file.
+	grep -v 'locale' $(ADD-ON)/jar.mn > $(GIT_EXPORT_DIR)/jar.mn
+	# Don't load our copy of loop.properties, load the browser's one instead.
```

Comment:
It's not obvious to me why this is necessary.... is it so that a beta add-on can be installed at the same time as (and correctly override) the system add-on?  If so, that's worth a comment.

Or is it just part of the way the export ended up needing to be structured?